packages/botonic-nlp/tests/tasks/text-classification/process/processor.test.ts

```diff
@@ -0,0 +1,39 @@
+import { Processor } from '../../../../src/tasks/text-classification/process/processor'
+import * as helper from '../../../helpers/tasks/text-classification/tools-helper'
+
+describe('Text Classification Processor', () => {
+  const processor = new Processor(
+    helper.preprocessor,
+    helper.tokensEncoder,
+    helper.classEncoder
+  )
+  test('Sample Processing', () => {
+    const sut = processor.process([
```

Comment:
sut is Subject Under Test (eg. the class instance you're testing). In this case the processor.